docroot/profiles/custom/acquia_onboard_jamesaldred_platform/acquia_onboard_jamesaldred_platform.info.yml

```diff
@@ -0,0 +1,36 @@
+name: Acquia Onboard jamesaldred Platform
+type: profile
+description: 'Onboarding Example'
+version: 8.x
+core: 8.x
+base profile: lightning
+excluded_dependencies:
+  - lightning_api
+  - lightning_layout
+  - lightning_media
+  - lightning_workflow
+  - lightning_contact_form
+  - lightning_page
+  - lightning_roles
+  - lightning_search
+  - lightning_landing_page
+  - lightning_media_audio
+  - lightning_media_bulk_upload
+  - lightning_media_document
+  - lightning_media_image
+  - lightning_media_instagram
+  - lightning_media_slideshow
+  - lightning_media_twitter
+  - lightning_media_video
+  - lightning_scheduler
+install:
+	- shield
+	- sitemap
+	- robotstxt
+	- pathauto
+	- default_content
+	- acsf
+	- trailing_slash
+themes:
+  - bartik
+  - seven
```

Comment:
@jamesaldred Needs to have an extra enter.

Reply:
@wouteradem this has now been corrected.